src/step.rs

```diff
@@ -119,6 +119,8 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
             // Defined to do nothing. These are added by optimization passes, to avoid changing the
             // size of MIR constantly.
             Nop => {}
+
+            InlineAsm { .. } => return Err(EvalError::InlineAsm),
```

Comment:
Yeah, it's been bugging me as an `Rvalue`. Arguably it should be a call but multiple outputs without touching the stack is a bit hard to model.